lua/catppuccin/groups/integrations/lsp_saga.lua

```diff
@@ -24,42 +25,89 @@ function M.get()
+		DiagnosticHead = { fg = C.blue },
 		CallHierarchyIcon = { fg = C.mauve },
 		CallHierarchyTitle = { fg = C.blue },
 		SagaShadow = { bg = C.crust },
 		OutlineIndent = { fg = C.overlay2 },
 
-		-- Because kind never worked after changing colorscheme
-		LspSagaWinbarFile = { fg = C.text },
-		LspSagaWinbarModule = { fg = C.blue },
-		LspSagaWinbarNamespace = { fg = C.peach },
-		LspSagaWinbarPackage = { fg = C.mauve },
-		LspSagaWinbarClass = { fg = C.mauve },
-		LspSagaWinbarMethod = { fg = C.mauve },
-		LspSagaWinbarProperty = { fg = C.sky },
-		LspSagaWinbarField = { fg = C.teal },
-		LspSagaWinbarConstructor = { fg = C.blue },
-		LspSagaWinbarEnum = { fg = C.green },
-		LspSagaWinbarInterface = { fg = C.peach },
-		LspSagaWinbarFunction = { fg = C.mauve },
-		LspSagaWinbarVariable = { fg = C.blue },
-		LspSagaWinbarConstant = { fg = C.sky },
-		LspSagaWinbarString = { fg = C.green },
-		LspSagaWinbarNumber = { fg = C.green },
-		LspSagaWinbarBoolean = { fg = C.peach },
-		LspSagaWinbarArray = { fg = C.blue },
-		LspSagaWinbarObject = { fg = C.peach },
-		LspSagaWinbarKey = { fg = C.red },
-		LspSagaWinbarNull = { fg = C.red },
-		LspSagaWinbarEnumMember = { fg = C.green },
-		LspSagaWinbarStruct = { fg = C.mauve },
-		LspSagaWinbarEvent = { fg = C.mauve },
-		LspSagaWinbarOperator = { fg = C.green },
-		LspSagaWinbarTypeParameter = { fg = C.green },
-		LspSagaWinbarTypeAlias = { fg = C.green },
-		LspSagaWinbarParameter = { fg = C.blue },
-		LspSagaWinbarStaticMethod = { fg = C.peach },
-		LspSagaWinbarMacro = { fg = C.red },
+		-- Lspkind icons support
+		LspKindClass = { fg = C.yellow },
+		LspKindConstant = { fg = C.peach },
+		LspKindConstructor = { fg = C.sapphire },
+		LspKindEnum = { fg = C.yellow },
+		LspKindEnumMember = { fg = C.teal },
+		LspKindEvent = { fg = C.yellow },
+		LspKindField = { fg = C.teal },
+		LspKindFile = { fg = C.rosewater },
+		LspKindFunction = { fg = C.blue },
+		LspKindInterface = { fg = C.yellow },
+		LspKindKey = { fg = C.red },
+		LspKindMethod = { fg = C.blue },
+		LspKindModule = { fg = C.blue },
+		LspKindNamespace = { fg = C.blue },
+		LspKindNumber = { fg = C.peach },
+		LspKindOperator = { fg = C.sky },
+		LspKindPackage = { fg = C.blue },
+		LspKindProperty = { fg = C.teal },
+		LspKindStruct = { fg = C.yellow },
+		LspKindTypeParameter = { fg = C.maroon },
+		LspKindVariable = { fg = C.peach },
+		LspKindArray = { fg = C.peach },
+		LspKindBoolean = { fg = C.peach },
+		LspKindNull = { fg = C.yellow },
+		LspKindObject = { fg = C.yellow },
+		LspKindString = { fg = C.green },
+		-- ccls-specific icons.
+		LspKindTypeAlias = { fg = C.green },
+		LspKindParameter = { fg = C.blue },
+		LspKindStaticMethod = { fg = C.peach },
+		-- Microsoft-specific icons.
+		LspKindText = { fg = C.green },
+		LspKindSnippet = { fg = C.mauve },
+		LspKindFolder = { fg = C.blue },
+		LspKindUnit = { fg = C.green },
+		LspKindValue = { fg = C.peach },
+	}
+end
+
+function M.custom_kind()
```

Comment:
So there's no need for user to manually add this to their configuration

Reply:
But if we set it this way colors are limited to `winbar` _only_, and there would be no highlight in other places that also utilizes these icons like "lspsaga outline".

Reply:
Ah yeah sorry I overlooked the outline feature